src/Calculator/Views/GraphingCalculator/EquationInputArea.xaml.cpp

```diff
@@ -173,17 +168,11 @@ void EquationInputArea::ReloadAvailableColors(bool isHighContrast)
         return;
     }
 
-    // Use a blank brush to clear out the color before setting it. This is needed because going
```

Comment:
The issue was due to the brush not changing when you use a different High Contrast color set (only the property SolidColorBrush.Color was updated). Using a Color instead of a Brush fixes this issue